examples/cloud-backup-snapshot/snapshot.json

```diff
@@ -21,10 +21,10 @@
         }
       }
     },
-    "CloudProviderSnapshots": {
-      "Type": "MongoDB::Atlas::CloudProviderSnapshots",
+    "BackupSnapshot": {
+      "Type": "MongoDB::Atlas::BackupSnapshot",
```

Comment:
Update name to "MongoDB::Atlas::CloudBackupSnapshot" here and throughout the resource